### PR TITLE
[ALLUXIO-1791] Improve javadoc in AlluxioFuseUtils

### PR DESCRIPTION
**Pull request by qzhong0605**
There should be newlines before the @param sections. We follow this convention in the rest of the codebase. I just add a newline before the @param sections.

### CODE REVIEW

**Comment by gpang** [comment]
@qzhong0605 Thanks for submitting a pull request to Alluxio!

We found your commits on on this PR aren't linked to a valid Github account yet (see your commits at https://github.com/Alluxio/alluxio/pull/2998/commits). 
For us to properly credit your contributions, could you please set up your Github account by following the Github instructions here?

Thank you again for your contribution to Alluxio!

**Comment by qzhong0605** [comment]
I have updated my Github account with add my email now. This is my failure. Are there any questions?

File: core/client/src/main/java/alluxio/client/file/BaseFileSystem.java [hunk 1]
```diff
@@ -284,6 +284,7 @@ public void setAttribute(AlluxioURI path, SetAttributeOptions options)
     FileSystemMasterClient masterClient = mContext.acquireMasterClient();
     try {
       masterClient.setAttribute(path, options);
+      LOG.info("Setted file " + path.getPath() + " attribute to " + options.toString());
```
Comment (calvinjia) [inline core/client/src/main/java/alluxio/client/file/BaseFileSystem.java]: `Setted` => `Set`, also you shouldn't need the `toString()` call.

Comment (qzhong0605) [inline core/client/src/main/java/alluxio/client/file/BaseFileSystem.java]: I have make a change the code. Are there any problems?

**Comment by yupeng9** [comment]
@qzhong0605 it seems your git history is corrupted. Could you create a new PR?

**Comment by qzhong0605** [comment]
I have create a new PR. There are problems on my git repo.

**Pull request closed without merging**